spec/factories/kudos.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :kudo do
+    title { 'title' }
+    content { 'content' }
+    giver_id { '1' }
+    receiver_id { '2' }
```

Comment:
Placing hardcoded values as an ids of associated objects may cause some issues. 
What will happen, when you won't have an employee with id 1 or 2 in your tests? 

Factory Bot offers solution for that - it can create all associated objects automatically:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations

so you can have something like:

```suggestion
    giver factory: :employee
    receiver factory: :employee
```

Reply:
I was wondering about it but I did not find a better solution than typing in hard code :)

Additionally, I wanted to ask because to pass rubocop I added two exceptions:

RSpec / ExampleLength:
   Exclude:
       - spec / ** / *

RSpec / MultipleExpectations:
   Exclude:
       - spec / ** / *

is it OK?

Reply:
As regards rubocop in my opinion it is fine, but it would be good to get Oskars opinion on it as well :)

Do you have any questions how the associations work in the factory?